.github/workflows/publish-package.yml

```diff
@@ -0,0 +1,65 @@
+name: publish distributions
+
+on:
+  push:
+    branches:
+    - main
+    tags:
+      - v*
+  pull_request:
+    branches:
+    - main
+  release:
+    types: [published]
+  workflow_dispatch:
+
+jobs:
+  build-and-publish:
+    name: Build and publish Python distro to (Test)PyPI
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+
+    - name: Install build and twine
+      run: |
+        python -m pip install --upgrade pip setuptools wheel
+        python -m pip install build twine
```

Comment:
I'm not also installing `check-manifest` here as the `prune **` strategy from PR #68 (which I like as it is simple and easy to read) doesn't work with `check-manifest` at this time (c.f. https://github.com/mgedmin/check-manifest/issues/144)